### PR TITLE
Update changelog for 9.2.0

### PR DESCRIPTION
**Pull request by Chaoba**
This pr update changelog for 9.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
  - Fixed a crash when calling the `Style.removeImage` method with the name of a nonexistent image. ([#16391](https://github.com/mapbox/mapbox-gl-native/pull/16391))
  - Fixed a crash when encountering an invalid polyline. ([#16409](https://github.com/mapbox/mapbox-gl-native/pull/16409))
  - Certain logging statements no longer run on the main thread. ([#16325](https://github.com/mapbox/mapbox-gl-native/pull/16325))
+ - Fixed library load failure on x86 devices. ([#368](https://github.com/mapbox/mapbox-gl-native-android/pull/368))
```
Comment (chloekraw) [inline CHANGELOG.md:27]:
```suggestion
 - Worked around a crash that could be triggered by SoLoader. ([#368](https://github.com/mapbox/mapbox-gl-native-android/pull/368))
```

Comment (chloekraw) [reply CHANGELOG.md:27]: Does https://github.com/mapbox/mapbox-gl-native-android/pull/368/ only affect x86 devices? I didn't see that in the PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
...
+ - Fixed a crash caused by JobIntentService in telemetry on devices with API > 26. ([#472](https://github.com/mapbox/mapbox-events-android/pull/472)
```
Comment (chloekraw) [inline CHANGELOG.md:28]:
```suggestion
 - Worked around crashes caused by `JobIntentService` on devices >= API 26 by using a network request to report errors instead. ([#472](https://github.com/mapbox/mapbox-events-android/pull/472)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
...
 
 ## 9.1.0
```
Comment (chloekraw) [inline CHANGELOG.md:30]: add date we published the last stable release

File: CHANGELOG.md (hunk 1 above)
Comment (chloekraw) [reply CHANGELOG.md:27]: I'm also thinking these two new entries fit better in a new sub-section "Other changes", what do you think?

### Other changes

Comment (Chaoba) [reply CHANGELOG.md:27]: Yeah, only affect x86 devices.
Other changes sounds good.

File: CHANGELOG.md [hunk 4]
```diff
@@ -25,7 +25,11 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
  - Fixed a crash when encountering an invalid polyline. ([#16409](https://github.com/mapbox/mapbox-gl-native/pull/16409))
  - Certain logging statements no longer run on the main thread. ([#16325](https://github.com/mapbox/mapbox-gl-native/pull/16325))
 
-## 9.1.0
+### Other changes
+ - Fallback to `System.libraryLoader` if SoLoader fails to load the native library. ([#368](https://github.com/mapbox/mapbox-gl-native-android/pull/368))
```
Comment (chloekraw) [inline CHANGELOG.md:29]: Oh, you can add “...on x86 devices.” at the end of this sentence then!

**Pull request merged**